### PR TITLE
Contact form section

### PR DESCRIPTION
**Pull request by mckent05**
In this PR, we (My coding partner and I) have added a contact form to the mobile version of my portfolio project. These PR includes:

- A HTML file containing a form that was used to include various input fields to the project
- A CSS file that was used in styling the form added
- A linter file that was used to check for and correct linter errors
- A read me file that contains a detailed description of the project so far.

Please note: The live demo link is not included because I have not gotten to the part of the Microverse curriculum that teaches how to implement that.

### CODE REVIEW

File: mobile.html [hunk 1]
```diff
@@ -273,7 +305,50 @@ <h2 class="project-title3">Multi-Post Stories</h2>
...
+    <section class="contact-form">
+        <div class="contact-me">
+            <h2 class="contact-us">contact us</h2>
+            <h2 class="chat">let's chat</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="https://formspree.io/f/xjvjloyy" method="POST" class="form">
+          <input id="input" type="text" required placeholder="text input" maxlength="30" name="name">
+          <input type="email" required placeholder="myemail@gmail.com" name="email" id="email">
+          <textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" required></textarea>
```
Comment (icebox827) [inline mobile.html]: - [ ] **REQUIRED:** Your work is admirable, but there are some missing requirements here. Please add the maxlength property for the message and set it to 500

File: mobile.css [hunk 2]
```diff
@@ -1410,3 +1410,272 @@ button:hover {
...
+  height: 28px;
+  left: 118px;
+  bottom: 1%;
+  background-color: #7d73fc;
+  color: #fff;
+  text-transform: capitalize;
+  font-family: 'Roboto', sans-serif;
+  justify-content: center;
+}
+
```
Comment (icebox827) [inline mobile.css:1559]: - [ ] **REQUIRED:** It will be nice to have interactions for the button such as hover, active, disable and enable.

**Review by icebox827 — CHANGES_REQUESTED** [review]
Hi Tope, 

Good job so far!
Your project looks good, but there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] **REQUIRED:** It will be nice to add the Getting Started section in your README file, to indicate how to use your project.
- [ ] **REQUIRED:** The live demo link is really important, it helps us to see how your project is displayed and behaved. Please consider adding it in your README file.

_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Tope Akinlade, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**